getting-started/download.md

```diff
@@ -1,43 +1,70 @@
-| [WolvenKit](https://github.com/WolvenKit/WolvenKit/)                          | [![GitHub release (latest by date)](https://img.shields.io/github/v/release/WolvenKit/WolvenKit)](https://github.com/WolvenKit/WolvenKit/releases/latest)                                   | ![GitHub branch checks state](https://img.shields.io/github/workflow/status/WolvenKit/WolvenKit/check-only)    |
-
-The latest GitHub WolvenKit release will be marked on the left column with green text. Expand the **Assets** tab to view available downloads for any given WolvenKit release.
+The recommended download is the WolvenKit Nightly installer
+
+<table>
+  <tr align="left">
+    <th>Source Code</th>
+    <th>Installer</th>
+    <th>Checks</th>
+  </tr>
+  <tr>
+    <td>
+      <a href="https://github.com/WolvenKit/WolvenKit-nightly-releases/">
+        WolvenKit Nightly
+      </a>
+    </td>
+    <td>
+      <a href="https://github.com/WolvenKit/WolvenKit-nightly-releases/">
+        <img src="https://img.shields.io/github/v/release/WolvenKit/WolvenKit-nightly-releases" alt="GitHub nightly release (latest by date)">
+      </a>
+    </td>
+    <td>
+      <img src="https://img.shields.io/github/workflow/status/WolvenKit/WolvenKit/WolvenKit-Nightly" alt="GitHub branch checks">
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <a href="https://github.com/WolvenKit/WolvenKit/">
+        WolvenKit Stable
+      </a>
+    </td>
+    <td>
+      <a href="https://github.com/WolvenKit/WolvenKit/releases/latest">
+        <img src="https://img.shields.io/github/v/release/WolvenKit/WolvenKit" alt="GitHub stable release (latest by date)">
+      </a>
+    </td>
+    <td>
+      <img src="https://img.shields.io/github/workflow/status/WolvenKit/WolvenKit/check-only" alt="GitHub branch checks">
+    </td>
+  </tr>
+</table>
 
 {% embed url="https://www.nexusmods.com/cyberpunk2077/mods/2201" %}
 
-### WolvenKit
+## System Requirements
 
-The main WolvenKit application in a portable zip file (Recommended)
+* 64-bit Windows 10/11
+* .NET 6.0 (x64)
 
 ### WolvenKit Installer
 
-The main WolvenKit application in a Microsoft installer package
+The main WolvenKit application in a Microsoft installer package.<br />
+<a href="https://wiki.redmodding.org/wolvenkit/wolvenkit-app/home">User guides</a>
 
 ### WolvenKit Console
 
-The standalone command-line based community modkit for Cyberpunk (Originally CP77 Tools)
+The standalone command-line based community modkit that replaced CP77 Tools.<br />
+<a href="https://wiki.redmodding.org/wolvenkit/wolvenkit-cli/wolvenkit-console">User guides</a>
 
 ### Source Code
 
-A snapshot of the code used to compile this specific WolvenKit release. Only useful for developers.
+Only useful for developers, the source code is a snapshot used to compile the WolvenKit installer and WolvenKit console.
```

Comment:
This is a comma splice. Also unnecessary/inaccurate as there is no installer.

Reply:
I'm not dying on a hill over a word, it can be called MickeyMouseLovesPeanuts for all I care :D I just want to make the instructions clearer and reduce confusion for new modders. The initial setup of WolvenKit adds %USERPFILE% roaming files and while the Plugin installs aren't technically a part of WolvenKit application, that process does add programs outside of wherever WolvenKit is extracted to. Unless someone is in the know, such as a developer, it looks and smells like its installing things.